scripts/deployment-configs/2022-12-14-test1.js

```diff
@@ -0,0 +1,7 @@
+/**
+ * These are the deployment config values for test staking pool deployment on 2022-12-14
+ * This pool should be deployed to EWC
+ */
+
+export const CLAIM_MANAGER_ADDRESS = "";
+export const PATRON_ROLE = "email.roles.verification.apps.energyweb.auth.ewc";
```

Comment:
I think this export could be useful too

```suggestion
export const INITIATOR_ADDRESS = ""
export const PATRON_ROLE = "email.roles.verification.apps.energyweb.auth.ewc";
```